german/the-black-hack.md

```diff
@@ -46,9 +46,9 @@ Rüstung bietet Schutz, indem sie allen erlittenen Schaden reduziert. Jedes Stü
 
 |     Typ        | Punkte |
 |:--------------:|:------:|
-|  Festes Wams   |   2    |
-|  Lederrüstung  |   4    |
-|   Kettenhemd   |   6    |
+| Gambeson       |   2    |
```

Comment:
Ist vielleicht nur ein persönlicher Tick, aber für mich ist "Gambeson" so ein typisches "DSA-Wort", das niemand kennt, bevor er mit dem Rollenspiel anfängt, dass von 1000 Leuten, die auf der Straße gefragt wurden, keiner kennt und das nicht im Duden steht.

Ich habe beim Übersetzen irgendwie einen Widerwillen, ein Fremd- oder Lehnwort zu verwenden, wenn ein deutscher Begriff existiert (und "Gambeson" ist auch im Englischen eher obskur und stammt wohl ursprünglich aus Frankreich, in Deutschland heißt das Ding historisch "Sarrock"). Auch wenn womöglich der deutsche Begriff nicht so geläufig ist - z.B. denke ich, dass heute der Begriff "Level" für das entsprechende Regelkonzept bei der Charakterentwicklung *sehr* viel verbreiteter ist, als "Stufe". Ich kann (aus gar keinem guten Grund) aber mit "Stufe" trotzdem besser schlafen.